src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -37,7 +37,7 @@ param()
 
 function New-AzConnectedKubernetes {
     [OutputType([Microsoft.Azure.PowerShell.Cmdlets.ConnectedKubernetes.Models.Api20240715Preview.IConnectedCluster])]
-    [CmdletBinding(DefaultParameterSetName = 'CreateExpanded', PositionalBinding = $false, SupportsShouldProcess, ConfirmImpact = 'Medium')]
+    [CmdletBinding(DefaultParameterSetName = 'CreateExpanded', PositionalBinding = $false, ConfirmImpact = 'Medium')]
```

Comment:
Noting for the record that we removed `SupportsShouldProcess` because this cmdlet has NEVER supported this and claiming we do triggers an error from `Invoke-ScriptAnalyser`.  Someone might want to add support for this later, but that should be a different story.